boards/FSM/Src/Io/Io_SoftwareWatchdog.c

```diff
@@ -13,8 +13,6 @@ void Io_HardwareWatchdog_Refresh(void)
 
 void Io_SoftwareWatchdog_TimeoutCallback(SoftwareWatchdogHandle_t watchdog)
 {
-    BREAK_IF_DEBUGGER_CONNECTED();
-
```

Comment:
Can you add this back for now? Want to do more testing to determine if this is really a problem before we commit to removing it.